sysprep/activate_instance.ps1

```diff
@@ -264,18 +264,16 @@ function Verify-ActivationStatus {
 
   [bool]$active = $false
   [String]$activation_status = $null
-  [String]$status = $null
 
   try {
-    $slmgr_status = & cscript //E:VBScript //nologo $env:windir\system32\slmgr.vbs /dli
+    $activation_status = (Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform\Activation" -Name ProductActivationResult).ProductActivationResult
   }
   catch {
     return $active
   }
 
-  $status = $slmgr_status | Select-String -Pattern '^License Status:'
-  # The initial space is to ensure "Unlicensed" does not match.
-  if ($status -match ' Licensed') {
+  # Anything other than 0x0 is a failure.
```

Comment:
I'd suggest keeping both method. 

If the regkey matches don't do the string match.
if the rekey doesn't match, do the string match attempt as a fall through.

Reply:
Moved this to a new PR cause the old branch was so old it was better to just rebase a new one
https://github.com/GoogleCloudPlatform/compute-image-windows/pull/252